Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -17,38 +17,78 @@
 from cil.optimisation.algorithms import Algorithm
 from cil.optimisation.functions import IndicatorBox
 from numpy import inf
+import numpy
 import warnings
 
-class SIRT(Algorithm):
+class SIRT(Algorithm):    
 
-    r'''Simultaneous Iterative Reconstruction Technique
+    r"""Simultaneous Iterative Reconstruction Technique 
+
+    Simultaneous Iterative Reconstruction Technique (SIRT) solves
+    the following problem
+
+    .. math:: A x = b
+
+    The SIRT algorithm is 
+
+    .. math:: x^{k+1} =  \mathrm{proj}_{C}( x^{k} + D ( A^{T} ( M * (b - Ax) ) ) ),
+
+    where :math:`M = A*\mathbb{1}`, :math:`D = A^{T}\mathbb{1}`, :math:`\mathbb{1}` is a :code:`DataContainer` of ones
```

Comment:
The comments say M = A^T*ones, same for D, but shouldn't it be 1 divided by that?

Reply:
fixed  

```
    where :math:`M = \frac{1}{A*\mathbb{1}}`, :math:`D = \frac{1}{A^{T}\mathbb{1}}`, :math:`\mathbb{1}` is a :code:`DataContainer` of ones
    and :math:`\mathrm{prox}_{C}` is the projection over a set :math:`C`.
```